Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -553,7 +553,7 @@ def bottle_formula(f, args:)
           "prefix"   => bottle.prefix,
           "cellar"   => bottle_cellar.to_s,
           "rebuild"  => bottle.rebuild,
-          "date"     => Pathname(filename.to_s).mtime.strftime("%F"),
+          "date"     => Pathname(fn).mtime.strftime("%F"),
```

Comment:
```suggestion
          "date"     => Pathname(filename).mtime.strftime("%F"),
```

but given `filename` was already being used here this might clash?

Reply:
Yeah, `filename` was assigned to mean something else elsewhere in this method.